ios/Classes/SwiftFlutterStarPrntPlugin.swift

```diff
@@ -265,11 +257,26 @@ public class SwiftFlutterStarPrntPlugin: NSObject, FlutterPlugin {
                 } else {
                     builder.appendQrCodeData((command["appendQrCode"] as! String).data(using: encoding), model: qrCodeModel, level: qrCodeLevel, cell: cell)
                 }
-            } else if (command["appendBitmap"] != nil) {
+            }else if let imageData = (command["appendBitmapImg"] as? FlutterStandardTypedData)?.data {
```

Comment:
There is already exist `appendBitmapByteArray` command in master branch which does the same thing.

Reply:
Removed and committed